src/include/sof/audio/component_ext.h

```diff
@@ -172,9 +170,7 @@ static inline int comp_copy(struct comp_dev *dev)
 
 	/* copy only if we are the owner of the component */
 	if (cpu_is_me(dev->ipc_config.core)) {
-		perf_cnt_init(&dev->pcd);
 		ret = dev->drv->ops.copy(dev);
-		perf_cnt_stamp(&dev->pcd, comp_perf_info, dev);
```

Comment:
This I think we have to keep, so can you surround these with "ifdef __ZEPHYR__"?

Reply:
I was thinking to move them to the caller later, anyway, let's use #ifdef guard to reduce risk for now.